website/www/site/layouts/shortcodes/flink_java_pipeline_options.html

```diff
@@ -16,7 +16,6 @@
 Use generatePipelineOptionsTableJava and generatePipelineOptionsTablePython respectively.
 Should be called before running the tests.
 -->
-<div class="table-container-wrapper">
```

Comment:
Would you mind fixing these unrelated diff? I also saw this weird diff before, this generated html changes depends on the platform.